hack/generate-operator-bundle.py

```diff
@@ -0,0 +1,149 @@
+#!/usr/bin/env python3
```

Comment:
this file should not be needed anymore

Reply:
As I said multiple times, not 100% true

Reply:
I don't really understand why we were working in the `make bundle` approach then. But if you want to keep this and work on that later, I'm fine with it